com.openwall.John.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "com.openwall.John",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "john",
+    "tags": [ "1.9J1-4bf9652", "1.9.0 Jumbo 1" ],
```

Comment:
Tags are not used here on flathub.

Reply:
I like them. Should I remove?

Reply:
Well, they are not used anyway. So they should be removed here